### DISCUSSION TITLE
HRRR Fields/Levels - Sfc/Nat/Prs

### DISCUSSION THREAD

**Discussion by tlg7** [NOAA-GSL/GSL-Regional-Model-Forum#34] [Q&A] [answered]
When downloading HRRR data (specifically SBCAPE and 0-6km shear components), I noticed that those variables are available for download under surface field output, native grid output, and pressure grid output. 

Each appears to be exactly the same, but I wanted to inquire why a variable like SBCAPE or 0-6 km shear would be included in three different field types? I just want to make sure there is no reason to pick from one category over the other.

**Comment by EricJames-NOAA** [comment, marked as answer]
@tlg7 thanks for the question.  For the HRRR, all the 2D fields are also included in the 3D (native level and isobaric) files, so that users can just download one type of file for their application.  So the fields are identical, and you can use them from whichever file is easiest.

**Reply by tlg7** [reply]
Hi Eric, my apologies for the late reply, but thank you for that explanation. Using surface fields for my case has worked perfectly well.

**Comment by mldubey** [comment]
Follow up on this. I have found an important case for my research in which the prs and sfc differ, and I would like to know why.
When looking at U and V above 500 mbar (at 300 and 250 mbar), the results from sfc contain more small scale structure that is not present in the prs data.
Is this an artifact or noise, or is this smaller scale structure actually there and the prs is smoothing out the sfc above 500 mbar?
<img width="928" alt="Screenshot 2025-06-26 at 11 08 23" src="https://github.com/user-attachments/assets/c013a601-2944-488e-9aec-d0df3fca3535" />
<img width="946" alt="Screenshot 2025-06-26 at 11 08 30" src="https://github.com/user-attachments/assets/39dfd570-f5ef-44c6-b7c4-7eb34a9e16ae" />
<img width="948" alt="Screenshot 2025-06-26 at 11 09 00" src="https://github.com/user-attachments/assets/76508590-c720-402d-ac9b-84dc6ab91c65" />
![Spectra_Compare_SFCPRS](https://github.com/user-attachments/assets/48588298-f206-415e-befc-cb974cc4fbfc)

**Reply by mldubey** [reply]
Note that the resulting spectra using prs has a much higher effective resolution than the sfc data (rolls off around 100 km as opposed to ~20 km).

**Reply by EricJames-NOAA** [reply]
@mldubey thanks for the question.  In the HRRR, there is some smoothing of certain fields in the prs files that is not present in the other files.  This is a legacy thing stemming from use of the pressure level files by NWS forecasters, who find some amount of smoothing helpful for their displays.  

So if you are interested in the native 3-km structure of the fields, we recommend using the native level or surface files.

**Reply by mldubey** [reply]
Thanks so much!
